tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -106,71 +118,93 @@ class TelemetryValidatorTest(absltest.TestCase):
   #   telemetry_validator.TelemetryValidator({}, 1, TimeoutCallback)
   #   threading.Timer(2, lambda: self.assertIsTrue(timeout))
 
+  def testTelemetryValidatorGetErrorsReturnsAllErrors(self):
```

Comment:
can you please leave a blank line between the arrange, act, assert parts in the tests